centos-7/build-openssl111d.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+cd /root
+git clone https://github.com/openssl/openssl.git
```

Comment:
Can save bandwidth by doing `git clone --single-branch`. Or could just download the OpenSSL release .tar.gz from their website.

Reply:
It was downloading the `.tar.gz` before, but I was running into build errors involving `zlib`, even with the `no-comp` option passed to `./config`. For now I just decided to switch to the source, but I'll add a TODO to look into this apparent discrepancy.